homepairs/homepairsUI/src/Screens/Components/LocationItem/LocationItem.tsx

```diff
@@ -4,6 +4,7 @@ import * as BaseStyles from 'homepairs-base-styles';
 import { HomePairFonts } from 'res/fonts.web';
 
 type LocationItemProps = {
+    testID?: string,
```

Comment:
change from testID to idTest (and then change to whatever you rename idTest to (see earlier comment)